docs/source/rpc.rst

```diff
@@ -0,0 +1,42 @@
+.. role:: hidden
```

Comment:
It looks like these titles show up in the sidebar:

```
Distributed RPC Framework
RPC and RRef Framework
Distributed Autograd Framework
```

Shouldn't we only have `Distributed RPC Framework` in the sidebar?

Reply:
This is now fixed.

Reply:
This is fixed in the latest update.